docs/Reference/API-Methods.md

```diff
@@ -2786,7 +2788,38 @@ None
         ]
     }
     ```
+### eth_submitWork
+
+Submits a Proof of Work (Ethash) solution to the blockchain.
+
+Used by mining software such as [Ethminer](https://github.com/ethereum-mining/ethminer).
+
+**Parameters**
+
+* DATA, 8 Bytes - The retrieved nonce.
```

Comment:
The retrieved nonce. -> Retrieved nonce.

Reply:
Done